### DISCUSSION TITLE
115所有文件突然都无法正常访问

### DISCUSSION THREAD

**Discussion by szhyf** [AlistGo/alist#3926] [Q&A]
版本3.14正常使用过程中，突然所有文件都无法正常访问，可以正常列目录，任何文件都无法打开
日志提示如下：
�[31mERRO�[0m[2023-03-23 10:16:50] {"status":403,"message":"no cookie value","request_id":"641BB69264A57639335C6E38"}
[GIN] 2023/03/23 - 10:16:50 | 200 | 328.093152ms | 192.168.1.239 | GET "*********"
Error https://github.com/alist-org/docs/issues/1: http: wrote more than the declared Content-Length

这是115升级了还是Alist参数调整了？

**Comment by qinguoluo** [comment]
昨天晚上都在正常使用。你是不是从其他端登录过115网盘，比如说网页、电脑版、手机版，然后alist端被顶出来了？

**Reply by szhyf** [reply]
> 昨天晚上都在正常使用。你是不是从其他端登录过115网盘，比如说网页、电脑版、手机版，然后alist端被顶出来了？

列目录都正常的，说明已经成功登录115，TOKEN没问题。只是抓取文件就失败了，看错误日志貌似是AList的问题

**Reply by smarxtboy** [reply]
2023年3月23日，使用Alist挂载115网盘，发现之前没有出现的问题是：成功挂载后，网盘的资源无法下载，可以上传。目前115网盘自身的客户端没有出现这样的问题。其次这个问题导致配合Infuse 已经无法观看网盘任何电影电视剧。官方回答是不支持第三方挂载。

**Comment by Terry-Wing** [comment]
跟你的问题一样，今天115不行了，能读取得到文件和目录，但是打不开

**Reply by dandyl1** [reply]
我也是。今天凌晨5点多还正常，睡一觉起来不行了。我还打电话问115客服是不是限速了，他说本来没限速，在虚拟环境下用不正常跟他们没关系，在线看不了的建议把视频下载下来看。

**Reply by dandyl1** [reply]
限制成了龟速

**Reply by szhyf** [reply]
> 我也是。今天凌晨5点多还正常，睡一觉起来不行了。我还打电话问115客服是不是限速了，他说本来没限速，在虚拟环境下用不正常跟他们没关系，在线看不了的建议把视频下载下来看。

我8点多还可以用，到10点就不行了

**Reply by dandyl1** [reply]
看来是分批部署的。看看能不能解决吧，本来还想看个115永久会员，幸亏没开。好不了run阿里云了～

**Reply by szhyf** [reply]
> 看来是分批部署的。看看能不能解决吧，本来还想看个115永久会员，幸亏没开。好不了run阿里云了～

应该是Alist的问题，有人回退3.13就可以用了

**Reply by dandyl1** [reply]
是吗。我用CloudDrive也不行，以为是115的问题。我rollback一下试试

**Reply by dandyl1** [reply]
兄弟应该是115搞了啥操作，CloudDrive2推送了一个更新，我更新了一下115速度正常了。你可以先下一个CloudDrive2来用，等Alist更新

**Comment by szhyf** [comment]
> 版本3.14正常使用过程中，突然所有文件都无法正常访问，可以正常列目录，任何文件都无法打开 日志提示如下： �[31mERRO�[0m[2023-03-23 10:16:50] {"status":403,"message":"no cookie value","request_id":"641BB69264A57639335C6E38"} [GIN] 2023/03/23 - 10:16:50 | 200 | 328.093152ms | 192.168.1.239 | GET "*********" Error https://github.com/[alist-org/docs/issues/1](https://github.com/alist-org/docs/issues/1): http: wrote more than the declared Content-Length
> 
> 这是115升级了还是Alist参数调整了？

回滚到3.13可以用了，确定是3.14不知道啥变了

**Reply by SheltonZhu** [reply]
修复代码已提交

**Reply by szhyf** [reply]
> 修复代码已提交

大佬神速，感谢，等新版本

**Comment by smarxtboy** [comment]
2023年3月23日，使用Alist挂载115网盘，发现之前没有出现的问题是：成功挂载后，网盘的资源无法下载，可以上传。目前115网盘自身的客户端没有出现这样的问题。其次这个问题导致配合Infuse 已经无法观看网盘任何电影电视剧。官方回答是不支持第三方挂载。

**Comment by smarxtboy** [comment]
目前Alist回退到v3.12.2 解决了文件不能下载和读取的问题。

**Reply by zxkke** [reply]
我也是碰到一样的问题，能上传，能读取目录，但是不能读取文件，还以为是账号被封了，吓了我一跳

**Comment by zxkke** [comment]
找到对应内核的musl版本解压得到alist文件，alist文件上传到/usr/bin/，修改好alist文件的运行权限，上传的时候要把alist关掉



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年3月24日(周五) 晚上6:06
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [alist-org/alist] 115所有文件突然都无法正常访问 (Discussion #3926)




 
您好 请问版本回退应该如何操作 我没搜到教程（环境是路由器）
